<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-G2SQ9GRV" ignored="0" name="Test" tests="2" failures="2" timestamp="17 Nov 2018 16:19:07 GMT" time="0.053" errors="0">
  <testcase name="failConsciously" time="0.014" classname="package1.AssertStmt">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at package1.AssertStmt.failConsciously(AssertStmt.java:27)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- failConsciously -->
  <testcase name="positiveDigit" time="0.002" classname="package1.AssertStmt">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at package1.AssertStmt.positiveDigit(AssertStmt.java:21)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- positiveDigit -->
</testsuite> <!-- Test -->
